src/Registry/TypeRegistry.php

```diff
@@ -1253,7 +1253,8 @@ public function register_mutation( $mutation_name, $config ) {
 
 		$output_fields = [
 			'clientMutationId' => [
-				'type' => 'String',
+				'type'        => 'String',
+				'description' => __( 'The ID for the mutation output field', 'wp-graphql' ),
```

Comment:
We should probably update this description to match that of the input too? 🤔 

```suggestion
				'description' => __( 'If a \'clientMutationId\' input is provided to the mutation, it will be returned as output on the mutation. This ID can be used by the client to track the progress of mutations and catch possible duplicate mutation submissions.', 'wp-graphql' ),
```